### DISCUSSION TITLE
Translation bot says I should not squash

### DISCUSSION THREAD

**Discussion by jcserracampos** [reactjs/reactjs.org-translation#247] [Team Posts]
Bot is saying that I should not squash commits when it create a new PR, but my repository only allows squashing.
How should our team proceed?

Example: https://github.com/reactjs/pt-BR.reactjs.org/pull/264

**Comment by armujahid** [comment]
Merge option should always be available if you have write access to repo. Check this https://help.github.com/en/articles/about-merge-methods-on-github. Merge option requires no other permission unlike rebase or squash merging. Can you share the screenshot of the issue?

**Comment by jcserracampos** [comment]
@armujahid 
![Screen Shot 2019-07-23 at 15 11 50](https://user-images.githubusercontent.com/7038498/61736326-34721900-ad5c-11e9-988f-deafa15a0468.png)

**Comment by cezaraugusto** [comment]
@jcserracampos it should be allowed now
